salt/modules/zypper.py

```diff
@@ -2206,22 +2322,27 @@ def download(*packages, **kwargs):
     )
 
 
-def list_downloaded():
+def list_downloaded(root=None):
     '''
     .. versionadded:: 2017.7.0
 
     List prefetched packages downloaded by Zypper in the local disk.
 
+    root
+        operate on a different root directory.
+
     CLI example:
 
     .. code-block:: bash
 
         salt '*' pkg.list_downloaded
     '''
-    CACHE_DIR = '/var/cache/zypp/packages/'
+    cache_dir = '/var/cache/zypp/packages/'
+    if root:
+        cache_dir = os.path.join(root, cache_dir)
 
     ret = {}
-    for root, dirnames, filenames in salt.utils.path.os_walk(CACHE_DIR):
+    for root, dirnames, filenames in salt.utils.path.os_walk(cache_dir):
```

Comment:
Constant still variable...

Reply:
Looks like that github is a bit inconsistent today, but as commented I am changing CACHE_DIR, so is not a constant anymore. I would find confusing something like this:

```
def my_func():
  A = 10
  if cond():
    A = A + 5
```

I change A, but the reader do not expect that I do that based on the uppercase. Changing the the case will fix the expectation.